test/controller/controller.spec.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var Angular = require('angular');
+var expect  = require('chai').expect;
+var Sinon   = require('sinon');
+
+require('angular-mocks');
+
+describe('create controller', function () {
+
+  var $controller;
+  var $q;
+  var $scope;
+  var Postcard;
+
+  beforeEach(Angular.mock.inject(function ($injector) {
+    $controller = $injector.get('$controller');
+    $q          = $injector.get('$q');
+    $scope      = $injector.get('$rootScope').$new();
+    Postcard    = $injector.get('Postcard');
+  }));
+
+  describe('create', function () {
+
+    beforeEach(function () {
+      $controller('CreateCtrl', {
+        $scope: $scope
+      });
+
+      $scope.$apply();
+    });
+
+    it('exposes the newly created postcard', function () {
+      var payload = { front: 'front', back: 'back' };
+      var postcard = { id: 'psc_id' };
+
+      Sinon.stub(Postcard, 'create').returns($q.resolve(postcard));
```

Comment:
i know this is probably what it said to do in miyagi, but since you are stubbing this function in each test in this describe block, you could probably create the stub in the `beforeEach` and then do the `returns` in each of the tests...but it is ok to leave as is for now! just a little note haha